src/ApiService/ApiService/Log.cs

```diff
@@ -105,14 +109,12 @@ class AppInsights : ILog {
             }
         }
 
-
         Dictionary<string, double>? copyMetrics = null;
         if (metrics is not null) {
             copyMetrics = new(metrics);
         }
         _telemetryClient.TrackException(ex, copyTags, copyMetrics);
-
-        Log(correlationId, $"{message} : {ex.Message}", SeverityLevel.Error, tags, caller);
+        Log(correlationId, $"[{message}] {ex.Message}", SeverityLevel.Error, tags, caller);
```

Comment:
the callstack made me wonder if it was `ex` that was `null` here

Reply:
it's string builder inside of message

I reproed

Reply:
(and then I'll be proven wrong ....)